### PR TITLE
possibility to load the base64 encoded authentication file

### LINKED ISSUE #59: Load service account (json) from classpath
Hi all,

Is it also possible (or in future) to load the service account json from classpath:
quarkus.google.cloud.service-account-location=serviceaccount.json

src/main/resources/serviceaccount.json

Thanks

### PR DESCRIPTION
**Pull request by felipesabadini**
Fixes #59

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ it depends on where your application runs (inside our outside Google Cloud Platf
 
 The current authentication flow is as follows:
 - Check the `quarkus.google.cloud.service-account-location` property, if it exists, use the service account file from this location.
+- Check the `quarkus.google.cloud.service-account-encoded-key` property, if it exists, use the service account file encoded in base64.
```
Comment (loicmathieu) [inline README.md:34]:
```suggestion
- Check the `quarkus.google.cloud.service-account-encoded-key` property, if it exists, use the service account base64 encoded content.
```

File: common/runtime/src/main/java/io/quarkiverse/googlecloudservices/common/GcpConfiguration.java [hunk 2]
```diff
@@ -23,6 +23,12 @@
     @ConfigItem
     public Optional<String> serviceAccountLocation;
 
+    /**
+     * Google Cloud service account file encoded in base64.
```
Comment (loicmathieu) [inline common/runtime/src/main/java/io/quarkiverse/googlecloudservices/common/GcpConfiguration.java:27]:
```suggestion
     * Google Cloud service account base64 encoded content.
```

File: integration-tests/main/src/main/resources/application.properties [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # You can set the following properties if you don't use the default credentials and project ID
 #quarkus.google.cloud.service-account-location=
+#quarkus.google.cloud.service-account-encoded-key=
```
Comment (loicmathieu) [inline integration-tests/main/src/main/resources/application.properties:3]: You should use the location or the encoded key.
Better to remove this comment.

**Review by loicmathieu — COMMENTED** [review]
Hi,
Thanks for providing this PR.
I made some small comment/doc suggestions but overall this looks good.

I'll need to test the branch locally before merging as there is no automated test for authentication. I'll keep you posted on my test.

**Comment by felipesabadini** [comment]
Hi,
I've already made the changes to the proposed suggestions

**Review by loicmathieu — APPROVED** [review]
Thanks for your contributions.
Changes looks OK.
I'll merge it as soon as I had the opportunity to test it locally.

**Comment by loicmathieu** [comment]
@all-contributors add @felipesabadini for code

**Pull request merged**